### PR TITLE
(core): update application attributes

### PR DESCRIPTION
**Pull request by icfantv**
* show all, specified application attributes from edit form on static
view of application attributes on config tab.

### CODE REVIEW

**Comment by icfantv** [comment]
Ignore the branch name, not sure what I was thinking.

@zanthrash @anotherchrisberry PTAL.
@danielpeach I ordered the static properties to match the edit application form fields, please let me know if this messes you guys up.

File: app/scripts/modules/core/application/config/applicationAttributes.directive.html [hunk 1]
```diff
@@ -16,11 +12,31 @@
...
+              Instance health
+            </dt>
+            <dd ng-if="vm.application.attributes.platformHealthOnly">
+              This application considers only cloud provider health when executing tasks
+            </dd>
+            <dd ng-if="vm.application.attributes.platformHealthOnlyShowOverride">
+              This application will show a health override option for each operation
+            </dd>
+            <dt>Instance Port</dt>
+            <dd>LOOK UP INSTANCE PORT (vm.application.attributes.instancePort || settings.SOMETHING)</dd>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/application/config/applicationAttributes.directive.html]: Looks like the Netflix override implementation needs to get copied up to here (instance port, health messages)

Comment (icfantv) [inline app/scripts/modules/core/application/config/applicationAttributes.directive.html]: whooops!  placeholders only work when you actually see them.

**Comment by icfantv** [comment]
fixed HTML and broken test.

File: app/scripts/modules/core/application/config/applicationAttributes.directive.html [hunk 2]
```diff
@@ -16,11 +12,31 @@
...
+            <dt
+              ng-if="vm.application.attributes.platformHealthOnly || vm.application.attributes.platformHealthOnlyShowOverride">
+              Instance health
+            </dt>
+            <dd ng-if="vm.application.attributes.platformHealthOnly">
+              This application considers only cloud provider health when executing tasks
+            </dd>
+            <dd ng-if="vm.application.attributes.platformHealthOnlyShowOverride">
+              This application will show a health override option for each operation
+            </dd>
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/application/config/applicationAttributes.directive.html]: Don't you want to use the message you set on the controller i.e. `{{vm.healthMessage}}`, like you're doing in `app/scripts/modules/netflix/application/applicationAttributes.directive.html` below?

**Comment by icfantv** [comment]
fixed for reals this time

**Comment by anotherchrisberry** [comment]
LGTM

**Pull request merged**